src/ZEO/asyncio/client.py

```diff
@@ -547,7 +547,7 @@ def verify(self, server_tid):
     def get_peername(self):
         return self.protocol.get_peername()
 
-    def call_async_threadsafe(self, future, method, args):
+    def call_async_threadsafe(self, future, _, method, args):
```

Comment:
I prefer to use ``_`` to indicate an unused variable.  This is a common pattern and even built in to some languages.